config/job.yaml.tim

```diff
@@ -39,9 +44,25 @@ spec:
         volumeMounts:
         - mountPath: /opt/screwdriver
           name: screwdriver
+        - mountPath: /var/run/sd
+          name: logs
         env:
         - name: SD_TOKEN
           value: "{{token}}"
+      - name: logservice
+        image: golang:1.7
```

Comment:
I see. The job-tools(init) container is still solely responsible for mounting the log-service binary into the volume, right? In which case I'm assuming the image for the log-service container needs only to use Alpine and apply the glibc hack as the job-tools image does.